File: netCoreAPITest/src/MovieService/Samp.Movie.API/Models/Dtos/MovieDto.cs
using Newtonsoft.Json;
using Samp.Movie.Database.Enums;

namespace Samp.Movie.API.Models.Dtos
{
    public class MovieDto
    {
        [JsonProperty("id")]
        public string Id { get; set; }

        [JsonProperty("title")]
        public string Title { get; set; }

        [JsonProperty("runtimeminutes")]
        public int RuntimeMinutes { get; set; }

        [JsonProperty("startyear")]
        public int StartYear { get; set; }

        [JsonProperty("description")]
        public string Description { get; set; }

        [JsonProperty("rating")]
        public RatingDto Rating { get; set; }

        [JsonProperty("type")]
        public MovieType Type { get; set; }

        [JsonProperty("usdprice")]
        public decimal UsdPrice { get; set; }

        [JsonProperty("itemdatabase")]
        public string ItemDatabase { get; set; }
    }
}

File: netCoreAPITest/src/Samp.Web.MovieStore/ClientApp/src/models/responses/movie/movie.dto.ts
import { RatingDto } from "./rating.dto";

export interface MovieDto {
  id: string;
  title: string;
  runtimeminutes: number;
  startyear: number;
  description: string;
  rating: RatingDto;
  type: string;
  usdprice: number;
  itemdatabase: string;
}
